.travis.yml

```diff
@@ -21,5 +21,5 @@ deploy:
   script:    
     - heroku container:release web --app $HEROKU_APP
   on:
-    branch: master
+    branch: module9
```

Comment:
Don't forget to switch this back before merging

Reply:
Dane